packages/wonder-blocks-grid/components/gutter.js

```diff
@@ -20,41 +15,40 @@ import FixedWidthCell from "./fixed-width-cell.js";
  * grid sizes. If you specify the `small`, `medium`, or `large`
  * props then the component will only be shown at those grid sizes.
  */
-class Gutter extends React.Component<{
+export default class Gutter extends React.Component<{
     /** Should this gutter be shown on a Small Grid? */
-    small?: boolean,
+    small: boolean,
     /** Should this gutter be shown on a Medium Grid? */
-    medium?: boolean,
+    medium: boolean,
     /** Should this gutter be shown on a Large Grid? */
-    large?: boolean,
-    /**
-     * The size of the media layout being used. Populated by MediaLayoutWrapper.
-     * @ignore
-     */
-    mediaSize: MediaSize,
-    /**
-     * The current media layout spec being used. Populated by MediaLayoutWrapper.
-     * @ignore
-     */
-    mediaSpec: MediaSpec,
+    large: boolean,
+    /** Should this gutter be shown at Medium or larger grids? */
+    mdOrLarger: boolean,
+    /** Should this gutter be shown at Medium or smaller grids? */
+    mdOrSmaller: boolean,
 }> {
     static defaultProps = {
-        small: false,
-        medium: false,
-        large: false,
+        small: true,
+        medium: true,
+        large: true,
+        mdOrLarger: true,
+        mdOrSmaller: true,
```

Comment:
Again, feels overly complex and easy to get into a contradictory state, but perhaps I am misreading this.